### DISCUSSION TITLE
Startup with multiple tabs and directories

### DISCUSSION THREAD

**Discussion by johnvandermolen** [PowerShell/PowerShell#19084] [Q&A]
Hi all,

I’m John and i’m new in this community and relatively new with powershell.
How can I startup powershell with multiple tabs in different directories?

Thanks!!

**Comment by rhubarb-geek-nz** [comment]
Hi. On which platform? PowerShell is a command line tool and doesn't really know about windows with tabs. If you are using Microsoft Terminal I would look at that for how you can have multiple programs open in the same window in different tabs.

The current directory is an attribute of a Windows process not a thread, so I would expect a single PowerShell process can only have one current directory even if running multiple threads. To have multiple current directories you need multiple instances of PowerShell running.
